sdks/java/io/snowflake/src/test/java/org/apache/beam/sdk/io/snowflake/test/unit/write/SnowflakeIOWriteTest.java

```diff
@@ -144,6 +155,11 @@ public void writeToExternalWithKVInput() {
                 .withSnowflakeService(snowflakeService));
 
     pipeline.run(options).waitUntilFinish();
+
+    List<String> actualData = FakeSnowflakeDatabase.getElements(FAKE_TABLE);
+    List<String> testDataInStrings =
+        testData.stream().map(e -> e.toString()).collect(Collectors.toList());
```

Comment:
Is it possible to make it like this : `Long::toString()` ?

Reply:
Let me check

Reply:
It was poosible to change it:
` testData.stream().map(Object::toString).collect(Collectors.toList());`
Is it ok?

Reply:
It is oke :D